source/_integrations/tis_control.markdown

```
@@ -0,0 +1,27 @@
+ha_domain: tis_control
+ha_platforms:
+  - switch
+ha_codeowners:
+  - '@karimtis'
+  - '@Mustafa744'
+
+ha_integration_type: integration
+---
+
+The TIS Control integration makes it possible to integrate and control TIS Control devices within Home Assistant.
+For full list of the products visit https://www.tiscontrol.com/.
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)